packages/fiori/src/themes/sap_horizon_dark/ShellBar-parameters.css

```
@@ -1,10 +1,33 @@
 :root {
+	--_ui5_shellbar_root_height: 3.25rem;
 	--_ui5_shellbar_logo_outline_color: var(--sapContent_FocusColor);
 	--_ui5_shellbar_logo_outline: var(--sapContent_FocusWidth) var(--sapContent_FocusStyle) var(--_ui5_shellbar_logo_outline_color);
+	--_ui5_shellbar_logo_outline_offset: -0.125rem;
 	--_ui5_shellbar_outline_offset: -0.125rem;

```

Should be done for Evening too:

```suggestion
	--_ui5_shellbar_outline_offset: -0.1875rem;
```